### PR TITLE
feat: Update zksync era sdk introduction page

### PR DESCRIPTION
**Pull request by Ngozi-Txfusion**
# What

- Updated the content of the SDK home page to improve clarity and usability.
- Enhanced the look and feel of the home page with UI design improvements.
- Increased the maximum line length for markdown files from 120 to 300 characters.
- Changed the template source URL from github:matter-labs/docs-nuxt-template#add-husky to github:matter-labs/docs-nuxt-template to fix the 404 error.
`ERROR  Failed to download https://api.github.com/repos/matter-labs/docs-nuxt-template/tarball/add-husky: 404 Not Found at download (node_modules/giget/dist/index.mjs:31:11) at process.processTicksAndRejections (node:internal/process/task_queues:95:5) at async downloadTemplate (node_modules/giget/dist/index.mjs:302:5) at async resolveConfig (node_modules/c12/dist/index.mjs:288:20) at async extendConfig (node_modules/c12/dist/index.mjs:241:21) at async loadConfig (node_modules/c12/dist/index.mjs:176:5) at async loadNuxtConfig (node_modules/@nuxt/kit/dist/index.mjs:2516:18) at async loadNuxt (node_modules/nuxt/dist/index.mjs:4284:19) at async loadNuxt (node_modules/@nuxt/kit/dist/index.mjs:2570:19) at async Object.run (node_modules/nuxi/dist/chunks/prepare.mjs:53:18) at async runCommand$1 (node_modules/nuxi/dist/shared/nuxi.9edf0930.mjs:1648:16) at async runCommand$1 (node_modules/nuxi/dist/shared/nuxi.9edf0930.mjs:1639:11) at async runMain$1 (node_modules/nuxi/dist/shared/nuxi.9edf0930.mjs:1777:7) [7:14:08 AM]  ERROR  Failed to download https://api.github.com/repos/matter-labs/docs-nuxt-template/tarball/add-husky: 404 Not Found error: postinstall script from "zksync-docs" exited with 1`

# Why
These changes were made to provide a better user experience, fix broken links, and ensure the code adheres to the latest standards and practices.

# Additionally:
- Due to the enforced line length, I couldn't send a GitHub commit, which resulted from the markdown linting.
- 'bun install' couldn't start because the '#add-husky' branch/tag was not found, so the URL was updated to point to the correct branch."

# Image

Screenshot 2024-05-28 at 07 15 18
Screenshot 2024-05-28 at 07 30 51

### CODE REVIEW

File: content/sdk/00.index.md [hunk 1]
```diff
@@ -1,21 +1,147 @@
...
-a high-performance scaling solution for Ethereum.
+## What is zkSync Era SDKs?
+
+zkSync Era SDKs help developers create decentralized applications (dApps) for the zkSync Era network. zkSync Era is a
+Layer 2 solution for Ethereum. It uses ZK rollup technology to provide scalable and low-cost transactions, keeping the
+security and trust of the Ethereum blockchain.
+
+<div align="center">
+    <img width="50%" src="/images/sdk/zksync-era-sdks.png" alt="zkSync Era SDKs"/>
+</div>
```
Comment (itsacoyote) [inline content/sdk/00.index.md:17]: We can keep this in here for now but we should change this later to use the markdown image syntax.
```
::div{class="text-center"}
![zkSync Era SDKs](/images/sdk/zksync-era-sdks.png){ width="50%"}
::
```

Right now the markdown image component has limited styling options which I'll have to edit for it to work.

File: content/sdk/00.index.md [hunk 2]
```diff
@@ -1,21 +1,147 @@
...
+target: _blank
+---
+Integrate zksync GO SDK for zkSync Era features.
+::
+::card
+---
+title: Python
+icon: i-heroicons-cog
+to: /sdk/python/getting-started
+target: _blank
```
Comment (itsacoyote) [inline content/sdk/00.index.md:64]: We should avoid making these `target: _blank` to open in a new tab. Recommended accessibility guidelines suggest opening within the same tab, especially for links that are internal.

File: content/sdk/00.index.md [hunk 3]
```diff
@@ -1,21 +1,147 @@
...
+
+Find the resources you need for assistance with our personalized help services.
+
+::card-group
+::card
+---
+title: Community
+icon: i-heroicons-users
+to: #
+target: _blank
```
Comment (itsacoyote) [inline content/sdk/00.index.md:112]: Same here with the `target: _blank`.

File: content/sdk/10.js/00.ethers/00.ethers.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+menuTitle: "JavaScript Versions"
+description: "Explore zkSync-ethers SDK versions"
+tags: "zksync, sdks, zkSync Era SDK, ethereum, era network, javascript versions"
+---
+
+These versions help developers interact with the zkSync Era blockchain. It covers most common
+operations developers require and provides a starting point for those new to the zkSync Era.
+
+::callout
+Take a look at [migration guide](/sdk/js/ethers/v6/migration) if you are migrating from `zksync-web3`.
```
Comment (itsacoyote) [inline content/sdk/10.js/00.ethers/00.ethers.md:12]:
```suggestion
Take a look at the [migration guide](/sdk/js/ethers/v6/migration) if you are migrating from `zksync-web3`.
```

File: content/sdk/10.js/00.ethers/00.ethers.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+- Node: >=v18 ([installation guide](https://nodejs.org/en/download/package-manager))
+
+### Installation
+
+To install SDK for zkSync Era, run the command below in your terminal.
+
+::code-group
+
+```bash [ethers-v5]
```
Comment (itsacoyote) [inline content/sdk/10.js/00.ethers/00.ethers.md:25]: Is there a reason this is `ethers-v5` with the dash vs `ethers v6` that doesn't?

File: content/sdk/10.js/00.ethers/00.ethers.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+to: /sdk/js/ethers/v5/getting-started
+target: _blank
+---
+Integrate JS with zkSync Era for secure transactions.
+::
+::card
+---
+title: V6
+to: /sdk/js/ethers/v6/getting-started
+target: _blank
```
Comment (itsacoyote) [inline content/sdk/10.js/00.ethers/00.ethers.md:52]: Same for these cards, all internal links should not have a `target: _blank`

File: layouts/sdk.vue [hunk 7]
```diff
@@ -13,7 +13,7 @@ const navigation = computed(() => nav!.value.find((item) => item._path.startsWit
         <UAside>
           <UNavigationTree
             :links="mapContentNavigation(navigation)"
-            default-open
+            default-open-all
```
Comment (itsacoyote) [inline layouts/sdk.vue:16]: Let's leave the links setting to default-open for now. It is harder to read the fully expanded navigation.

File: pages/sdk/[...slug].vue [hunk 8]
```diff
@@ -52,11 +52,10 @@ const links = computed(() =>
   ].filter(Boolean)
 );
 </script>
-
 <template>
   <UPage v-if="page">
     <UPageHeader
-      :title="page.title"
+      :title="page.menuTitle || page.title"
```
Comment (itsacoyote) [inline pages/sdk/[...slug].vue:58]: I like the idea, but it feels a little too confusing to have multiple title labels. For accessibility and consistency, we should have the link in the navigation match the title on the page. Let's leave the title to be 'Introduction' and drop the menuTitle for now.

**Review by itsacoyote — CHANGES_REQUESTED** [review]
I like how the page is looking! Just a few comments on some things to adjust.

File: pages/sdk/[...slug].vue (hunk 8 above)
Comment (Ngozi-Txfusion) [reply pages/sdk/[...slug].vue:58]: Done

File: layouts/sdk.vue (hunk 7 above)
Comment (Ngozi-Txfusion) [reply layouts/sdk.vue:16]: Done

File: content/sdk/10.js/00.ethers/00.ethers.md (hunk 6 above)
Comment (Ngozi-Txfusion) [reply content/sdk/10.js/00.ethers/00.ethers.md:52]: done

File: content/sdk/10.js/00.ethers/00.ethers.md (hunk 5 above)
Comment (Ngozi-Txfusion) [reply content/sdk/10.js/00.ethers/00.ethers.md:25]: edited

File: content/sdk/00.index.md (hunk 3 above)
Comment (Ngozi-Txfusion) [reply content/sdk/00.index.md:112]: done

File: content/sdk/00.index.md (hunk 2 above)
Comment (Ngozi-Txfusion) [reply content/sdk/00.index.md:64]: edited

File: content/sdk/00.index.md [hunk 9]
```diff
@@ -1,21 +1,147 @@
 ---
-title: zkSync Era SDKs
-description:
+title: "Home"
+menuTitle: "Introduction"
+description: "zkSync Era SDKs empower developers to seamlessly interact with the zkSync Era network."
+navigation: true
+tags: "zksync, sdks, zkSync Era SDK, ethereum, era network, javascript, go, swift, python, java, rust"
```
[the remark this answers is not in the document]
Comment (Ngozi-Txfusion) [reply content/sdk/00.index.md]: edited

File: content/sdk/00.index.md [hunk 10]
```diff
@@ -1,21 +1,147 @@
 ---
-title: zkSync Era SDKs
-description:
+title: "Home"
+menuTitle: "Introduction"
+description: "zkSync Era SDKs empower developers to seamlessly interact with the zkSync Era network."
+navigation: true
```
[the remark this answers is not in the document]
Comment (Ngozi-Txfusion) [reply content/sdk/00.index.md]: done

**Review by itsacoyote — APPROVED** [review]
🎉

**Pull request closed without merging**